spk/syncthing-inotify/src/dsm-control.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+# Package
+PACKAGE="syncthing-inotify"
+DNAME="Syncthing-Inotify"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SYNCTHING_DIR="/usr/local/syncthing"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+USER="syncthing"
+INOTIFY="${INSTALL_DIR}/bin/syncthing-inotify"
+CONFIG_DIR="${SYNCTHING_DIR}/var"
+INOTIFY_OPTIONS="-home=${CONFIG_DIR}"
+
+# Read additional startup options from /usr/local/syncthing-inotify/var/options.conf
+if [ -f ${INSTALL_DIR}/var/options.conf ]; then
+	source ${INSTALL_DIR}/var/options.conf
```

Comment:
Where is `options.conf` located? In the inotify package, or in the Syncthing package. Seems you're mixing the two up ;)